ServerMonitoringSystem.MessageProcessor/Persistence/MongoDbStatisticsRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Configuration;
+using MongoDB.Driver;
+using ServerMonitoringSystem.Shared.Domain;
+
+namespace ServerMonitoringSystem.MessageProcessor.Persistence;
+internal class MongoDbStatisticsRepository : IStatisticsRepository
+{
+    private readonly IMongoCollection<ServerStatistics> _statisticsCollection;
+
+    public MongoDbStatisticsRepository(IConfiguration configuration)
```

Comment:
lets move the db repo and config to a different project, and move the hardcoded configs to appsettings please